atlite/data.py

```diff
@@ -21,16 +24,20 @@ def get_features(cutout, module, features, tmpdir=None):
     Load the feature data for a given module.
 
     This get the data for a set of features from a module. All modules in
-    `atlite.datasets` all allowed.
+    `atlite.datasets` are allowed.
     """
     parameters = cutout.data.attrs
+    lock = SerializableLock()
     datasets = []
     get_data = datamodules[module].get_data
 
     for feature in features:
-        feature_data = get_data(cutout, feature, tmpdir=tmpdir, **parameters)
+        feature_data = get_data(cutout, feature, tmpdir=tmpdir, lock=lock, **parameters)
         datasets.append(feature_data)
 
+    if len(datasets) >= 1 and isinstance(datasets[0], Delayed):
+        datasets = dask.compute(*datasets)
```

Comment:
```suggestion
        datasets, = dask.compute(*datasets)
```
This lead to an error as `dask.compute` returns a tuple

Reply:
Doesn't lead to an error for me. Datasets are multiple datasets before and after.